GRAPHICS/EnableMoonCycleOverride.md

```diff
@@ -0,0 +1,30 @@
+---
+ns: GRAPHICS
+aliases: ["0x2C328AF17210F009", "_SET_EXTRA_TIMECYCLE_MODIFIER_STRENGTH"]
+---
+
+## ENABLE_MOON_CYCLE_OVERRIDE
+
+```c
+// 0x2C328AF17210F009
+void ENABLE_MOON_CYCLE_OVERRIDE(float phase);
+```
+
+Enable a custom moon cycle, allowing control of which lunar phase the moon is in.
+Valid values are from `0.0` to `1.0`, with `0.5` representing a full moon.
+
+- `0.1` Waxing Crescent
+- `0.2` First Quarter
+- `0.3` Waxing Gibbous
+- `0.5` Full Moon
+- `0.7` Waning Gibbous
+- `0.8` Third Quarter
+- `0.9` Waning Crescent
```

Comment:
Just an idea, but maybe would be great to add a table with better preview like [this](https://github.com/citizenfx/natives/pull/977#pullrequestreview-1830851887)?